README.rst

```diff
@@ -0,0 +1,162 @@
+
+.. image:: https://travis-ci.org/diffpy/diffpy.pdfmorph.svg?branch=master
+   :target: https://travis-ci.org/diffpy/diffpy.pdfmorph
+
+.. image:: http://codecov.io/github/diffpy/diffpy.pdfmorph/coverage.svg?branch=master
+   :target: http://codecov.io/github/diffpy/diffpy.pdfmorph?branch=master
+
+
+PDFmorph
+========================================================================
+
+PDFmorph is a Python software package designed to increase the insight 
+researchers can obtain from measured atomic pair distribution functions 
+(PDFs) in a model-independent way. The program was designed to help a 
+researcher answer the question: Has my material undergone a phase 
```

Comment:
seem to have lost the quotes around the  question.  It is a style issue, but I think I prefer the quotes, though I could be persuaded otherwise.

Reply:
I was curious about this, so I checked the MLA rules on it and found [this](https://style.mla.org/direct-question-in-a-sentence/). Let me know if you're still partial to the quotes, the meaning won't be lost either way!

Reply:
fixed